modules/ui.py

```diff
@@ -1174,6 +1175,9 @@ def create_ui(wrap_gradio_gpu_call):
             outputs=[init_img_with_mask],
         )
 
+    global  init_img_components
+    init_img_components = {"img2img":init_img, "inpaint":init_img_with_mask, "extras":extras_image}

```

Comment:
Proper whitespace..?
```suggestion
    global init_img_components
    init_img_components = {"img2img": init_img, "inpaint": init_img_with_mask, "extras": extras_image}
```